src/main/java/org/prebid/server/analytics/reporter/greenbids/model/GreenbidsUnifiedCode.java

```diff
@@ -0,0 +1,32 @@
+package org.prebid.server.analytics.reporter.greenbids.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Value;
+
+@Builder(toBuilder = true)
+@Value
+public class GreenbidsUnifiedCode {
+
+    @JsonProperty("value")
+    String value;
+
+    @JsonProperty("src")
+    Source source;
+
+    public enum Source {
+        gpidSource("gpid"),
+        storedRequestIdSource("storedRequestId"),
+        adUnitCodeSource("adUnitCode");
+
+        private final String value;
+
+        Source(String value) {
+            this.value = value;
+        }
+
+        public String getValue() {
+            return value;
+        }
+    }
+}
```

Comment:
please extract as a separate class

also, it's better to use the upper case with underscores to name the enum values, e.g. GPID_SOURCE